.circleci/config.yml

```diff
@@ -133,6 +133,20 @@ jobs:
       - <<: *attach_to_bootstrap
       - run: yarn test:integration
 
+  starters_publish:
```

Comment:
I'll add back the other jobs, but this is helpful to validate this approach and get feedback more quickly.

See [CircleCI](https://circleci.com/gh/DSchau/gatsby/48) for an example run.